src/dlgConnectionProfiles.cpp

```diff
@@ -1760,11 +1739,11 @@ bool dlgConnectionProfiles::validateProfile()
             if (!allowedChars.contains(name.at(i))) {
                 notificationAreaIconLabelWarning->show();
                 notificationAreaMessageBox->setText(
-                        QString("%1\n%2").arg(notificationAreaMessageBox->text(), tr("The %1 character is not permitted. Use one of the following:\n\"%2\".\n").arg(name.at(i), allowedChars)));
+                        QString("%1\n%2").arg(notificationAreaMessageBox->text(), tr("The %1 character is not permitted. Use one of the following:\"%2\".").arg(name.at(i), allowedChars)));
```

Comment:
I've been looking in this area myself recently and this simple concatenation of successive error messages may not be the best way of assembling multiple error/warning messages as they will like get run together (white-space merging) should the text be displayed as *rich* rather than plain text.

I'd suggest appending each separate error message as a `tr("<p>`...`</p>")` with arguments as needed into a `QStringList` and then putting the entirety of them, with a call to the `QStringList::join(QString())` method, into the `notificationAreaMessageBox` with the `setText(`...`)` method at the end of the `validateProfile()` assuming that it does not `return (bool);` before it gets to that point....